vizro-ai/docs/pages/tutorials/quickstart-dashboard.md

```diff
@@ -1,5 +1,9 @@
 # Dashboard generation
 
+!!! warning "Alpha Feature"
```

Comment:
```suggestion
!!! warning "This feature is under development"
```

Reply:
Or something similar I think, rather than "Alpha"

Reply:
To be more precise, it's not "under development" anymore. 😆 @maxschulz-COL wdyt?

Reply:
I would stick with "alpha" or maybe something like - "Feature currently in alpha version", because what is an alpha feature 😂 like a very dominant feature??